server/routes/probationPractitionerReferrals/myCasesPresenter.ts

```diff
@@ -0,0 +1,60 @@
+import { SentReferral, ServiceCategory } from '../../services/interventionsService'
+import PresenterUtils from '../../utils/presenterUtils'
+import { SortableTableHeaders, SortableTableRow } from '../../utils/viewUtils'
+
+export default class MyCasesPresenter {
+  constructor(private readonly sentReferrals: SentReferral[], private readonly serviceCategories: ServiceCategory[]) {}
+
+  readonly tableHeadings: SortableTableHeaders = [
+    { text: 'Referral', sort: 'none' },
+    { text: 'Service user', sort: 'ascending' },
+    { text: 'Service Category', sort: 'none' },
+    { text: 'Provider', sort: 'none' },
+    { text: 'Caseworker', sort: 'none' },
+    { text: 'Action', sort: 'none' },
+  ]
+
+  readonly tableRows: SortableTableRow[] = this.sentReferrals.map(referral => {
+    const { serviceCategoryId } = referral.referral
+    const serviceCategory = this.serviceCategories.find(aServiceCategory => aServiceCategory.id === serviceCategoryId)
+    if (serviceCategory === undefined) {
+      throw new Error(`Expected serviceCategories to contain service category with ID ${serviceCategoryId}`)
+    }
+
+    // i really want the actual names here, but that's an API call for each row - how can we improve this?
```

Comment:
Should we be storing the full name on the referral instead / as well? Again, might be a bit of work